libcore/tests/GOFFileTests/GOFFileTests.vala

```diff
@@ -153,7 +122,9 @@ void new_symlink_local_test () {
 
     file.query_update ();
     assert (file.info != null);
-    assert (file.get_symlink_target () == path);
+
+    string target = file.get_symlink_target ().dup ();
```

Comment:
you don't have to explicitly use .dup (), assigning an `unowned string` to a `string` should do it automatically

Reply:
Yes, now that I have corrected the .vapi for GOFFile, I guess you are right. I'll change it and check for any regressions.